021-IdentityForApps/Student/07-admin-graph.md

```diff
@@ -12,7 +12,7 @@ Leadership would like you to build an app that can query the B2C tenant and outp
 
 Luckily, your innovative developer is still here (they haven't taken any time off either) and they've built a **console app** that can do some of this (but you'll need to make some modifications). Luckily, the developer has parameterized the B2C bits, so you'll have to just make some updates to the configuration settings file (appsettings.json) in order to connect to your B2C tenant.
 
-**NOTE:** This DOTNETCORE console application is located in the folder named **MSGraphApp** in your Teams channel Files tab.
+**NOTE:** This DOTNETCORE console application is located in the **MSGraphApp** folder under the [student resources](Resources/MSGraphApp) or in your Teams channel Files tab.
```

Comment:
Please remove direct link to the WTH repo. Instead, refer to the relative path in the Resources.zip file provided by the coach.

Reply:
Fixed!